packages/feedback/src/SmileField.js

```diff
@@ -3,31 +3,42 @@ import PropTypes from 'prop-types';
 import { useField, useFormikContext } from 'formik';
 import FeedbackButton from './FeedbackButton';
 
-const btnStyles = { flex: 1, margin: '0 2% 0 2%' };
+const btnStyles = {};
+const groupStyles = { display: 'grid', gap: '1rem' };
 
 const SmileField = ({ name, options, onChange, autoFocusFeedbackButton }) => {
   const [{ value }] = useField(name);
   const { setFieldValue } = useFormikContext();
 
-  return options.map((option, i) => (
-    <FeedbackButton
-      autoFocus={i === 0 && autoFocusFeedbackButton}
-      style={btnStyles}
-      key={option.icon}
-      icon={option.icon}
-      iconSize="2x"
-      value={option}
-      active={value && value.icon}
-      onClick={() => {
-        setFieldValue(name, option);
-        if (onChange) {
-          onChange(option);
-        }
+  return (
+    <div
+      style={{
+        ...groupStyles,
+        gridTemplateColumns: `repeat(${options.length}, auto)`,
```

Comment:
Can we stick to [using Bootstrap for layout](https://getbootstrap.com/docs/4.0/layout/grid/) here? Just so that our components don't start to diverge too much from what the UI Kit uses before we decide on an upgrade path for everything

Reply:
From my brief search, Bootstrap 4 doesn't have a utility for `display: grid` or `gap`, which is really the right solution for the CSS here. It is bootstrap 4 that we're on, isn't it?

Reply:
Bootstrap has columns we should be [able to use](https://getbootstrap.com/docs/4.0/layout/grid/). Either that or [flex](https://getbootstrap.com/docs/4.0/utilities/flex/).

Do you think either of those will work?